app/controllers/course/material/controller.rb

```diff
@@ -2,8 +2,8 @@ class Course::Material::Controller < Course::ComponentController
   load_and_authorize_resource :folder, through: :course, through_association: :material_folders,
                                        class: Course::Material::Folder.name
 
-  before_action :check_component
-  before_action :add_folder_breadcrumb
+  before_action :check_component, unless: :material_show
```

Comment:
Oh yeah!  So in this case I would prefer to just present the attachments url in assessment show page.